### PR TITLE
(overwrite=false) parameter helps with bad internet connection

### PR DESCRIPTION
**Pull request by abit2**
it helps if you have a bad Internet connection :D

### CODE REVIEW

**Comment by Cadair** [comment]
The justification for having it set to true by default was that it allows us to update it and then makes it more likely that people will get the updated version. Instead of changing the default code snippet, could you mention the overwrite parameter in a sentence after?

File: doc/source/guide/acquiring_data/sample-data.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ download the sample data simply run the following command::
 
     import sunpy.data
     sunpy.data.download_sample_data()
-
+With a bad Internet connection it helps to set the overwrite=False parameter
```
Comment (dpshelio) [inline doc/source/guide/acquiring_data/sample-data.rst]: If you look at the [Travis errors](https://travis-ci.org/sunpy/sunpy/jobs/158358387#L1858), you find the following:
```
sample-data.rst:14: WARNING: Literal block ends without a blank line; unexpected unindent.
```

**Comment by wafels** [comment]
+1

**Comment by Cadair** [comment]
:+1:

**Pull request merged**